[Doc]: Update doc for "Set up the Documentation"

### Describe your documentation update

During the Step 3 from [README.rst](https://github.com/tomvothecoder/python-template-repo#readme), step for updating `docs/conf.py` needs to be added to avoid the following issue. 

```
>>> make docs
```

```
rm -rf docs/generated
cd docs && make html
Running Sphinx v5.3.0

Configuration error:
There is a programmable error in your configuration file:

Traceback (most recent call last):
  File "/Users/lee1043/mambaforge/envs/pcmdi_utils_dev/lib/python3.10/site-packages/sphinx/config.py", line 350, in eval_config_file
    exec(code, namespace)
  File "/Users/lee1043/Documents/Research/git/pcmdi_utils/docs/conf.py", line 24, in <module>
    import python_template
ModuleNotFoundError: No module named 'python_template'

make[1]: *** [html] Error 2
make: *** [docs] Error 2
```

--- a/README.rst
+++ b/README.rst
@@ -97,7 +97,7 @@ The library's documentation is setup with `Sphinx`_ and `Read the Docs Sphinx Th
 
 You just need to do a few things to get up and running:
 
-1. Update ``README.rst``, ``AUTHORS.rst``, ``HISTORY.rst``, and ``/docs/index.rst`` as needed
+1. Update ``README.rst``, ``AUTHORS.rst``, ``HISTORY.rst``, ``/docs/index.rst``, and ``/docs/conf.py`` as needed
 2. Decide how to deploy the documentation. This step is left up to you based on your needs.
    Follow the instructions in the provided links.
 

